text.cabal

```diff
@@ -166,6 +187,7 @@ library
     Data.Text.Internal.Lazy.Search
     Data.Text.Internal.Private
     Data.Text.Internal.Read
+    Data.Text.Internal.Reverse
```

Comment:
I'd prefer to move this new module under `other-modules`. This is way too "internal" to be exposed as a part of public API. @Lysxia what do you think?

Reply:
I'll do whatever you decide here.

Reply:
I've made it an other-module for now.